core/schema/module.go

```diff
@@ -791,36 +791,38 @@ func (s *moduleSchema) moduleToSchemaFor(ctx context.Context, module *core.Modul
 
 		newObjResolver := ObjectResolver{}
 		for _, field := range objTypeDef.Fields {
+			field := field
```

Comment:
Just a little bug waiting to happen! :beetle: 

Thankfully, all it was potentially doing was giving an incorrect error message, the behavior wasn't affected.